godtoolsTests/App/Share/Common/FileCache/FileCacheTests.swift

```diff
@@ -0,0 +1,253 @@
+//
+//  FileCacheTests.swift
+//  godtools
+//
+//  Created by Levi Eggert on 8/12/25.
+//  Copyright © 2025 Cru. All rights reserved.
+//
+
+import Testing
+@testable import godtools
+import Foundation
+
+@Suite(.serialized)
```

Comment:
Hey @rachaelblue here I run the tests serially instead of parallel since accessing the shared FileCache could cause unexpected results if running in parallel when adding and deleting the temp_directory. 

Some pretty good info on migrating from XCTests to Swift Tests (https://developer.apple.com/documentation/testing/migratingfromxctest)